docs/en/Mounting-Alluxio-FS-with-FUSE.md

```diff
@@ -1,104 +1,100 @@
+Alluxio-FUSE is based on the project [Filesystem in Userspace](http://fuse.sourceforge.net/) (FUSE), and most basic file system operations are supported. However, given the intrinsic characteristics of Alluxio, like its write-once/read-many-times file data model, the mounted file system will not have full POSIX semantics and will have specific limitations.  Please read the [section of limitations](#assumptions-and-limitations) for details.
 
 ## Requirements
 
-* Linux kernel 2.6.9 or newer
 * JDK 1.8 or newer
-* [libfuse](https://github.com/libfuse/libfuse) 2.9.3 or newer (2.8.3 has been reported to also work - with some warnings)
-
-## Building
-
-alluxio-fuse is only built with Alluxio when the `fuse` maven profile is active. This profile will be automatically activated by maven when it is detected that you are building Alluxio with a JDK version 8 or newer.
-
-For compatibility with Java 7, binary alluxio distributions may ship without alluxio-fuse support, so you will need to build your own Alluxio if you want to use alluxio-fuse on your deployment.
-
-The best way to do so is to either clone the Alluxio [GitHub repository](https://github.com/alluxio/alluxio) and choose your favourite branch from git, or to grab a [source distribution](https://github.com/alluxio/alluxio/releases) directly. Please, refer to [this page](Building-Alluxio-Master-Branch.html)) for building instructions.
+* [libfuse](https://github.com/libfuse/libfuse) 2.9.3 or newer (2.8.3 has been reported to also work - with some warnings) for Linux
+* [osxfuse](https://osxfuse.github.io/) 3.7.1 or newer for MacOS
 
 ## Usage
 
 ### Mount Alluxio-FUSE
 
 After having properly configured and started the alluxio cluster, and from the node where you wish to mount Alluxio, point a shell to your `$ALLUXIO_HOME` and run:
 
-{% include Mounting-Alluxio-FS-with-FUSE/alluxio-fuse-mount.md %}
-
-This will spawn a background user-space java process (alluxio-fuse) that will mount the file system on the specified `<mount_point>`. Note that `<mount_point>` must be an existing and empty path in your local file system hierarchy and that the user that runs the `alluxio-fuse.sh` script must own the mount point and have read and write permissions on it. Also note that, currently, you are limited to have only one Alluxio-FUSE mount per node.
+```bash
+$ integration/fuse/bin/alluxio-fuse mount mount_point [alluxio_path]
+```
 
-### Unmount Alluxio-FUSE
-
-To umount a previoulsy mounted Alluxio-FUSE file sytem, on the node where the file system is mounted, point a shell to your `$ALLUXIO_HOME` and run:
-
-{% include Mounting-Alluxio-FS-with-FUSE/alluxio-fuse-umount.md %}
+This will spawn a background user-space java process (alluxio-fuse) that will mount the alluxio path specified at `alluxio_path` to the local file system on the specified `mount_point`. 
```

Comment:
is this java process supposed to be long-running? (e.g., showing up in `top`)? otherwise, I would not expose this level of details to users "This will spawn a background user-space java process (alluxio-fuse) "